x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/index.tsx

```diff
@@ -40,19 +39,15 @@ const allRulesTabs = [
     id: AllRulesTabs.rules,
     name: i18n.RULES_TAB,
     disabled: false,
+    href: '/rules',
   },
   {
     id: AllRulesTabs.monitoring,
     name: i18n.MONITORING_TAB,
     disabled: false,
-  },
-  {
-    id: AllRulesTabs.exceptions,
-    name: i18n.EXCEPTIONS_TAB,
-    disabled: false,
+    href: '/rules/monitoring',
```

Comment:
should we use our constants?

Reply:
I think we can remove href, it is not in used.